runtime/codert_vm/jswalk.c

```diff
@@ -807,13 +808,15 @@ jitWalkResolveMethodFrame_walkD(J9StackWalkState *walkState, UDATA ** pendingSen
 		UDATA fpParmNumber = J9SW_JIT_FLOAT_ARGUMENT_REGISTER_COUNT - *floatRegistersRemaining;
 #ifdef J9VM_INTERP_STACKWALK_TRACING
 		if (walkState->flags & J9_STACKWALK_ITERATE_O_SLOTS) {
+			assert(NULL != walkState->jitFPRegisterStorageBase);
```

Comment:
This can be moved above the if check, then you won't need the second assert in this function.

use `Assert_CodertVM_true` instead of `assert` (no need to include the `assert.h` header as well).
same for other asserts in this file